src/examples/example.scala

```diff
@@ -20,7 +20,9 @@ def run(lang: String): Unit =
       es"Es ${number(1)} en español" &
       fr"C'est ${number(1)} en français"
 
+   val subset: Messages[Ru & Es] = msg.subset[Ru & Es]
```

Comment:
Maybe the problem is that I'm mixing up unions and intersections...

Reply:
Ups, I meant to use unions, so that should have fail, `Ru | Es` properly fails to compile as Ru is not in mylangs, `En | Es` narrow as expected.  Because I used intersection it wrongly compiled.  An slip of the mind because I think the product of messages should modelled by intersections, but I think unions are probably safer. By same past experience the duality in scalaland is not complete so proofs reversing <: to >: when flipping from union to intersection wouldn't be found by the compiler. Maybe things have improved but dunno, the intersection is not a proper product in scalac so I don't think the co-product duality can be safely assumed.